src/Components/UserList.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+const User = ({ name, image, repoCount }) => {
+  return (
+    <div>
+      <img src={image} alt="github profile picture" width={200} />
+
+      <p className='user-name'>
+        { name }
+      </p>
+
+      <p className="repo-count">
+        { repoCount }
+      </p>
+
+    </div>
+  )
+};
+
+const usersSortedByRepoCount = (users) => {
```

Comment:
I could change it to a verb that indicates the action that the method performs. Instead of `usersSortedByRepoCount`, I would rename to `sortUsersByRepoCount`